src/test/java/com/pgs/booking/service/FlightServiceTest.java

```diff
@@ -0,0 +1,162 @@
+import java.time.LocalDateTime;
+import java.time.Month;
+import java.util.List;
+import java.util.Optional;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Mockito.*;
+
+@ExtendWith(MockitoExtension.class)
+class FlightServiceTest {
+
+    @Mock
+    private FlightRepository flightRepository;
+
+    private final FlightDtoMapper flightDtoMapper = new FlightDtoMapper();
+    private final CreateUpdateFlightDtoMapper createUpdateFlightDtoMapper = new CreateUpdateFlightDtoMapper();
+
+    @InjectMocks
+    private FlightService underTest;
+
+    @BeforeEach
+    void setUp() {
+        underTest = new FlightService(flightRepository, flightDtoMapper, createUpdateFlightDtoMapper);
+    }
+
+    private static Flight FLIGHT = Flight.builder()
+            .id(1L)
+            .type(Flight.TypeOfFlight.ECONOMY)
+            .departureDate(LocalDateTime.of(2021, Month.DECEMBER, 9, 18, 30))
+            .arrivalDate(LocalDateTime.of(2021, Month.DECEMBER, 9, 23, 30))
+            .build();
+
+    private static Flight FLIGHT_1 = Flight.builder()
+            .id(1L)
+            .type(Flight.TypeOfFlight.BUSINESS)
+            .departureDate(LocalDateTime.of(2021, Month.JULY, 9, 11, 30))
+            .arrivalDate(LocalDateTime.of(2021, Month.JULY, 9, 14, 30))
+            .build();
+
+    private static FlightDto FLIGHT_DTO = FlightDto.builder()
+            .id(1L)
+            .type(Flight.TypeOfFlight.PREMIUM)
+            .departureDate(LocalDateTime.of(2021, Month.NOVEMBER, 12, 10, 30))
+            .arrivalDate(LocalDateTime.of(2021, Month.NOVEMBER, 12, 16, 30))
+            .build();
+
+    private static CreateUpdateFlightDto CREATE_UPDATE_FLIGHT_DTO = CreateUpdateFlightDto.builder()
+            .type(Flight.TypeOfFlight.BUSINESS)
+            .departureDate(LocalDateTime.of(2021, Month.SEPTEMBER, 10, 8, 30))
+            .arrivalDate(LocalDateTime.of(2021, Month.SEPTEMBER, 10, 11, 30))
+            .build();
+
+    @Test
+    void testGetFlights() {
+        given(flightRepository.findAll())
+                .willReturn(List.of(new Flight(), new Flight(), new Flight(), new Flight()));
+        //when
+        var flights = underTest.getFlights();
+        //then
+        Assertions.assertEquals(4, flights.size());
+        verify(flightRepository).findAll();
+    }
+
+    @Test
+    void testGetFlight() {
+        //given
+        long id = 1L;
+        given(flightRepository.findById(id))
+                .willReturn(java.util.Optional.ofNullable(FLIGHT_1));
+        //when
+        var flight = underTest.getFlight(id);
+        //then
+        Assertions.assertEquals(Flight.TypeOfFlight.BUSINESS, flight.getType());
+        Assertions.assertEquals(LocalDateTime.of(2021, Month.JULY, 9, 11, 30), flight.getDepartureDate());
+        Assertions.assertEquals(LocalDateTime.of(2021, Month.JULY, 9, 14, 30), flight.getArrivalDate());
```

Comment:
you should check against the properties of the object returned by `flightRepository`
```suggestion
        Assertions.assertEquals(FLIGHT.getType(), flight.getType());
        Assertions.assertEquals(FLIGHT.getDepartureDate(), flight.getDepartureDate());
        Assertions.assertEquals(FLIGHT.getArrivalDate(), flight.getArrivalDate());
```